### ISSUE TITLE
LogFileProtect = Y issue

### ISSUE THREAD

**Issue by frank-dittrich** [magnumripper/jtrTestSuite#65]
```
(master)test $ ./jtrts.pl -q -stoponerror -v -v -v
-------------------------------------------------------------------------------
- JtR-TestSuite (jtrts). Version 1.13, Dec 21, 2014.  By, Jim Fougeron & others
- Testing:  John the Ripper 1.8.0.6-jumbo-1-1584-g8ce5a1b [linux-gnu 64-bit AVX-ac]
--------------------------------------------------------------------------------

[...]
Execute john: ../run/john -ses=tst-  -pot=tst-.pot dynamic_0_tst.in --wordlist=pw.dic -form=dynamic_0 2>&1 >/dev/null


Cmd_data = 
ERROR: LogFileProtect enabled in john.conf, and tst-.log exists



Execute john: ../run/john -show   -pot=tst-.pot dynamic_0_tst.in -form=dynamic_0

d_show_line = 
0 password hashes cracked, 1502 left

[...]
```
So, the test suite should add `LogFileProtect=N` to the local conf file and/or delete `tst-.log`.

**Comment by frank-dittrich** [comment]
LogFileProtect=Disabled is what we need here.

**Comment by jfoug** [comment]
Easy to add to john-local.conf in the test directory.

**Comment by jfoug** [comment]
I believe this fixes your problem  794c29f     @frank-dittrich, please test.
